src/qibolab/transpilers/gate_decompositions.py

```diff
@@ -41,7 +41,9 @@ def count_1q(self, gate):
 
     def __call__(self, gate):
         """Decompose a gate."""
-        if gate.parameters or (gate.__class__ is gates.FusedGate):
+        if gate.__class__ is gates.I:
+            decomposition = self.decompositions[gate.__class__]
+        elif gate.parameters or (gate.__class__ is gates.FusedGate):
             decomposition = self.decompositions[gate.__class__](gate)
         else:
             decomposition = self.decompositions[gate.__class__]
```

Comment:
```suggestion
        decomposition = self.decompositions[gate.__class__]
        if callable(decomposition):
            decomposition = decompositions(gate)
```
or
```suggestion
        decomposition = self.decompositions[gate.__class__]
        if not isinstance(decomposition, list):
            decomposition = decompositions(gate)
```
should work for all gates, without having to check special cases for some gates.